galley/pkg/crd/validation/webhook.go

```diff
@@ -120,6 +120,12 @@ type WebhookParameters struct {
 
 	// Enable galley validation mode
 	EnableValidation bool
+
+	// ManageWebhookConfig determines whether the ValidatingWebhookConfiguration
+	// should be watched and updated by the webhook itself. This can be disabled,
+	// in which case we simply assume the configuration to be correct. This is
+	// helpful if you want to remove ClusterRole permissions from galley.
+	ManageWebhookConfig bool
```

Comment:
I'd add: `... remove ClusterRole permissions from galley and it is safe to do that e.g. if someone else is taking care of that`.
Wording may be bad, hopefully you get the idea